fix_admin_auto_mode.py

```
@@ -0,0 +1,178 @@
+            try:
+                portfolio_data = portfolio_response.json()
+                current_strategies = len(portfolio_data.get("active_strategies", []))
+                print(f"Current active strategies: {current_strategies}")
+            except ValueError:
+                print("Could not parse portfolio response")
+        else:
+            print(f"Portfolio check failed: {portfolio_response.status_code}")
+
+        # Step 3: Grant admin full strategy access if needed
+        if current_strategies < 3:
+            print(f"\nStep 3: Granting admin full strategy access...")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)